### PR TITLE
Book store2

### PR DESCRIPTION
**Pull request by arikarim**
Milestone 2 book-Store setup:

- [x] Implement `BookList and BookForm` container components.
- [x] Implement book component.
- [x] implement `addBook` and `removeBook` actions and reducers.
- [x] Finally create the redux store

### CODE REVIEW

File: src/containers/ BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        id="title"
+        type="text"
+        name="title"
+        placeholder="Title"
+        onChange={handleChange}
+        required
+      />
+      <select>
+        <option>1</option>
+      </select>
```
Comment (Torres-ssf) [inline src/containers/ BooksForm.js]: - [x] The BookForm should have a select element that displays all categories. The form functionality should not be working, for now, clicking on the submit button should not trigger any action at the current milestone

File: src/containers/BookList.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+// import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+// import Book from '../components/Book';
```
Comment (Torres-ssf) [inline src/containers/BookList.js]: - [x] Let's remove this commented code.

File: src/containers/BookList.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const mapStateToProps = (state) => ({
+  books: state,
+});
+
+// const mapDispatchToProps = (dispatch) => (
+//   bindActionCreators({
+//     removeBook,
+//   }, dispatch)
+// );
```
Comment (Torres-ssf) [inline src/containers/BookList.js]: - [x] Let's remove this commented code.

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
+import { createStore } from 'redux';
 import './index.css';
 import App from './App';
+import bookReducer from './reducers/book';
 import reportWebVitals from './reportWebVitals';
 
+// combineReducers({
+//   books: bookReducer,
+// });
+
```
Comment (Torres-ssf) [inline src/index.js]: - [x] Let's remove this commented code.

File: src/App.js [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
+import BooksForm from './containers/ BooksForm';
 
 function App() {
   return (
     <div className="App">
-      Hello Mr Code Reviewer
+      <BookList />
+      <BooksForm />
     </div>
   );
```
Comment (Torres-ssf) [inline src/App.js]: - [x] App component should be placed at the `src/components` dir

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi @arikarim :hand: 
Awesome job on this milestone.
We just need to make some changes before moving on.


- [x] We need to remove all commented code from the app

- [x] The src/reducers/index.js should combine all reducers and export them. For now, we will only have the book reducer. Later on, we will add the filter reducer as well.
```javascript
import { combineReducers } from 'redux'
import books from './books'

export default combineReducers({
  books
})
```

Happy coding :computer: 
**Sérgio Torres**

File: src/containers/ BooksForm.js (hunk 1 above)
Comment (arikarim) [reply src/containers/ BooksForm.js]: Done

File: src/containers/BookList.js (hunk 2 above)
Comment (arikarim) [reply src/containers/BookList.js]: Done

File: src/containers/BookList.js (hunk 3 above)
Comment (arikarim) [reply src/containers/BookList.js]: Done

File: src/App.js (hunk 5 above)
Comment (arikarim) [reply src/App.js]: Done

**Review by uche-inyama — APPROVED** [review]
## Approved 

### Hi Ari :slightly_smiling_face: 

We are good here, move on to the next milestone.

[**Observation**] There is a commented line of code in line 7 of src/index.js file, kindly remove it.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**